app/src/main-process/menu/build-default-menu.ts

```diff
@@ -297,11 +297,16 @@ export function buildDefaultMenu(
         click: emit('merge-branch'),
       },
       separator,
+      {
+        label: __DARWIN__ ? 'Compare on Desktop' : '&Compare on Desktop',
```

Comment:
The screenshot shows that local and remote functions are grouped with the separator above - does it make sense to move this up alongside `Update from Default Branch` and `Merge Branch`?